<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QFrame" name="frame_7">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../assets/icons/icons8-reserva-50.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text">
             <string>Inventário Doméstico</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="rowWrapPolicy">
             <enum>QFormLayout::DontWrapRows</enum>
            </property>
            <property name="labelAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="nomeLabel">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Nome</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="InputName">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>420</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="salaReaLabel">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Sala/Área</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="InputRoom">
              <property name="maximumSize">
               <size>
                <width>420</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="descriOLabel">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Descrição</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="InputDescription">
              <property name="maximumSize">
               <size>
                <width>420</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="marcaModeloLabel">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Marca/Modelo</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="InputBrand">
              <property name="maximumSize">
               <size>
                <width>420</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="dataDaCompraLabel">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Data Da Compra</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDateEdit" name="InputDate">
              <property name="maximumSize">
               <size>
                <width>420</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="maximumDate">
               <date>
                <year>2022</year>
                <month>12</month>
                <day>31</day>
               </date>
              </property>
              <property name="minimumDate">
               <date>
                <year>2022</year>
                <month>8</month>
                <day>15</day>
               </date>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
              <property name="date">
               <date>
                <year>2022</year>
                <month>8</month>
                <day>15</day>
               </date>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="valorDaCompraLabel">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Valor Da Compra</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="InputPrice">
              <property name="maximumSize">
               <size>
                <width>420</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="sRieLabel">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Série</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="InputSerie">
              <property name="maximumSize">
               <size>
                <width>420</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>25</number>
            </property>
            <item>
             <widget class="QPushButton" name="BtnAdd">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Adiciona</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../assets/icons/icons8-adicionar-coluna-50.png</normaloff>../assets/icons/icons8-adicionar-coluna-50.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BtnUpdate">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Atualizar</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../assets/icons/icons8-actualizar-50.png</normaloff>../assets/icons/icons8-actualizar-50.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BtnDelete">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Apagar</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../assets/icons/icons8-apagar-para-sempre-50.png</normaloff>../assets/icons/icons8-apagar-para-sempre-50.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="frameShape">
                <enum>QFrame::WinPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <family>Arial Black</family>
                    <pointsize>9</pointsize>
                    <weight>50</weight>
                    <italic>true</italic>
                    <bold>false</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                   </font>
                  </property>
                  <property name="text">
                   <string>Valor Total De Todos Itens</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="AlertValueTotal">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>R$ 100</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_4">
               <property name="frameShape">
                <enum>QFrame::WinPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="font">
                   <font>
                    <family>Arial Black</family>
                    <pointsize>9</pointsize>
                    <weight>50</weight>
                    <italic>true</italic>
                    <bold>false</bold>
                    <underline>false</underline>
                    <strikeout>false</strikeout>
                   </font>
                  </property>
                  <property name="text">
                   <string>Total De Itens</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="AlertTotalItens">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>10</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="Table">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <column>
          <property name="text">
           <string>Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nome</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sala/Área</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Descrição</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Marca/Modelo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Data Da Compra</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Valor Da Compra</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Série</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="maximumSize">
          <size>
           <width>5000</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
